tests/s-thread.c

```diff
@@ -21,7 +21,11 @@ static int b(void *arg)
 	return c(arg) + 1;
 }
 
-static int c(void *arg)
+#if __clang__
+__attribute__((optnone))
+#endif
+static int
+c(void *arg)
```

Comment:
This can be in the same line as before.

Reply:
This is modified by clang-format so we better just keep it.

Reply:
https://github.com/namhyung/uftrace/actions/runs/15950671790/job/44990175175

Reply:
Hmm.. ok.